person.rb

```diff
@@ -6,10 +6,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Also, as I mentioned in the book file, here you should also include a method for adding a rental, another `add_rental` method, so you can finish the setup for all associations. The code should look pretty similar to what I mentioned in the book file, just adjusting it to the person file, so please make sure to work on this issue.